services/director-v2/src/simcore_service_director_v2/modules/dask_clients_pool.py

```diff
@@ -80,6 +80,10 @@ async def _concurently_safe_acquire_client() -> DaskClient:
                         tasks_file_link_type = (
                             self.settings.COMPUTATIONAL_BACKEND_DEFAULT_CLUSTER_FILE_LINK_TYPE
                         )
+                    if cluster.type is ClusterTypeInModel.ON_DEMAND:
+                        tasks_file_link_type = (
+                            self.settings.COMPUTATIONAL_BACKEND_ON_DEMAND_CLUSTERS_FILE_LINK_TYPE
```

Comment:
Q: so you always have settings for all the `ClusterTypeInModel` types ready&valid in case the cluster type changes?

Reply:
yes

Reply:
that fix is not the best in the world, but will do for now regarding the 5GB issue.